src/ui/src/builder/builderManager.ts

```diff
@@ -406,6 +406,22 @@ export function generateBuilderManager() {
 		return state.value.logEntries;
 	};
 
+	const activeBlueprintRunId = computed(() => {
+		const logEntries = getLogEntries();
+		const runId =
+			logEntries.find((entry) => {
+				return !!entry.blueprintExecution;
+			})?.blueprintExecution?.runId ?? null;
+		const isActive = !logEntries
+			.filter((entry) => {
+				return entry?.blueprintExecution?.runId === runId;
+			})
+			.find( (entry) => {
+				return entry.exit
+			});
```

Comment:
It could be done in the same loop
```suggestion
			.filter((entry) => {
				return entry?.blueprintExecution?.runId === runId && entry.exit;
			});
```

Reply:
but is it more readable?
```
		const isActive = Boolean(
			logEntries
				.filter((entry) => {
					return entry?.blueprintExecution?.runId === runId
						&& !entry?.blueprintExecution?.exit;
				})?.[0]
		)
```

Reply:
I didn't mean to inline everything, it was just to avoid an extra loop on the array